services/venice-server/src/main/java/com/linkedin/venice/server/VeniceServer.java

```diff
@@ -204,19 +204,36 @@ private List<AbstractVeniceService> createServices() {
     jvmStats = new VeniceJVMStats(metricsRepository, "VeniceJVMStats");
 
     if (serverConfig.isSystemSchemaInitializationAtStartTimeEnabled()) {
+      String localControllerUrl = serverConfig.getLocalControllerUrl();
+      String d2ServiceName = serverConfig.getLocalControllerD2ServiceName();
+      String d2ZkHost = serverConfig.getLocalD2ZkHost();
+      String cluster = serverConfig.getSystemSchemaClusterName();
```

Comment:
Let's rename `cluster` to `systemStoreCluster` to avoid the confusion between the cluster that the server belongs to and the cluster that system schema locates in.